src/main/java/be/seeseemelk/mockbukkit/profile/PlayerProfileMock.java

```diff
@@ -191,25 +184,24 @@ public boolean complete(boolean textures, boolean onlineMode)
 	}
 
 	@Override
-	public boolean equals(Object obj)
+	public int hashCode()
 	{
-		if (this == obj) return true;
-		if (!(obj instanceof PlayerProfileMock otherProfile)) return false;
-		return Objects.equals(this.name, otherProfile.name) && this.uuid.equals(otherProfile.uuid) && this.properties.equals(otherProfile.properties);
+		return this.properties.hashCode();
 	}
 
 	@Override
-	public String toString()
```

Comment:
Why was the toString Method removed?

Reply:
It's just unnecessary

Reply:
Why though?

Reply:
Just no real reason to use it. Will add it back though.

Reply:
Good, we can't guarantee that people haven't used it so we wouldn't want to break compatibility